sonar-fsharp-plugin/src/main/java/org/sonar/plugins/fsharp/FSharpSonarWayProfile.java

```diff
@@ -32,63 +32,61 @@
  */
 package org.sonar.plugins.fsharp;
 
-import org.sonar.api.profiles.ProfileDefinition;
-import org.sonar.api.profiles.RulesProfile;
-import org.sonar.api.rules.Rule;
-import org.sonar.api.utils.ValidationMessages;
+import org.sonar.api.server.profile.BuiltInQualityProfilesDefinition;
 
-// based on plugins from from https://github.com/SonarSource
-public class FSharpSonarWayProfile extends ProfileDefinition {
+public class FSharpSonarWayProfile implements BuiltInQualityProfilesDefinition {
+
+  private static void activateRule(NewBuiltInQualityProfile profile, String ruleKey, String severity) {
```

Comment:
why `static``? Called from the instance method `define`, no access to static data needed.

Reply:
Ok i will update